app/(Components)/(Form)/BotzEvent/EventForm.tsx

```diff
@@ -150,9 +164,22 @@ export const EventForm: React.FC<{
 
 	const handleSubmit: React.FormEventHandler = (event) => {
 		event.preventDefault();
-		console.log('Hello there');
-	};
+		setLoading(true);
 
+		fetch('https://social-video-generatorz-server.cleverapps.io/BotzEvent', {
+			headers: {
+				'Content-Type': 'application/json',
+			},
+			method: 'POST',
+			body: JSON.stringify(data),
+		})
+			.then((res) => res.blob())
+			.then((blob) => {
+				const fileURL = URL.createObjectURL(blob);
+				setVideoUrl(fileURL);
+				setLoading(false);
+			});
+	};
```

Comment:
`.then`, `.then`, il manque un truc ? 👀 Genre si 💥

Reply:
Added catch and display an error under the generate button